python_modules/dagster/dagster_tests/core_tests/execution_tests/test_asset_backfill.py

```diff
@@ -712,6 +764,94 @@ def downstream_weekly_partitioned_asset(
     assert counts[2].num_targeted_partitions == 1
 
 
+def test_asset_backfill_status_count_with_backfill_policies():
```

Comment:
Should this be moved over to the other file?

Reply:
I originally thought this is very similar to the other "status count" test and wanted to keep it in the context here. However, it doesn't hurt to move it to the other file too so I just did.